packages/ddp-client/common/livedata_connection.js

```diff
@@ -1722,8 +1722,10 @@ export class Connection {
     self._outstandingMethodBlocks = [];
 
     self.onReconnect && self.onReconnect();
+
```

Comment:
Commit message should be updated (it's no longer `noRetryError`, etc).

Reply:
fixed